inst/@sym/isempty.m

```diff
@@ -60,7 +60,6 @@
 %! a = sym ([1 2]);
 %!assert (~isempty (sym (1)))
 %!assert (isempty (sym (se)))
-%!assert (isempty (se == []))
```

Comment:
that don't have any sense..., because [] is equal to []..., why it should return empty? (se is [])